api/tests/unit/organisations/chargebee/test_unit_chargebee_chargebee.py

```diff
@@ -287,8 +289,40 @@ def test_get_subscription_metadata(mocker, chargebee_object_metadata, addon_quan
     mocked_chargebee_cache.return_value.plans = {plan_id: chargebee_object_metadata}
     mocked_chargebee_cache.return_value.addons = {addon_id: chargebee_object_metadata}
 
+    return {
+        "customer_email": customer_email,
+        "mocked_chargebee_cache": mocked_chargebee_cache,
+        "mock_subscription_response": mock_subscription_response,
+    }
+
+
+@pytest.mark.parametrize("addon_quantity", (None, 1))
+def test_get_subscription_metadata(mocker, chargebee_object_metadata, addon_quantity):
+    # Given
+    status = "status"
+    plan_id = "plan-id"
+    addon_id = "addon-id"
+    subscription_id = "subscription-id"
+    customer_email = "test@example.com"
+
+    create_mocked_subscription_metadata(
+        mocker, chargebee_object_metadata, addon_quantity
+    )
     # When
-    subscription_metadata = get_subscription_metadata(subscription_id)
+    subscription = {
+        "status": status,
+        "id": subscription_id,
+        "plan_id": plan_id,
+        "addons": [
+            {
+                "id": addon_id,
+                "quantity": 2,
+                "unit_price": 0,
+                "amount": 0,
+            }
+        ],
+    }
```

Comment:
This should probably be in the `Given` section.

Reply:
Moved.